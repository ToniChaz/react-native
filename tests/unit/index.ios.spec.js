// index.ios.spec.js

import 'react-native'
import React from 'react'
import Index from '../../index.ios.js'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('Index component', () => {
	test('renders correctly', () => {
		const tree = renderer.create(
      <Index />
		)

		expect(tree.toJSON()).toMatchSnapshot()
	})
})

