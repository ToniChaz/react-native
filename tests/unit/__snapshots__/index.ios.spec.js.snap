// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index component renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Object {
        "flex": 1,
        "flexDirection": "column-reverse",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        pointerEvents="auto"
        style={
          Object {
            "backgroundColor": "#E9E9EF",
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "shadowColor": "black",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0.4,
            "shadowRadius": 10,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#204",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "height": 110,
                "transform": Array [
                  Object {
                    "rotate": "0deg",
                  },
                ],
                "width": 125,
              }
            }
          />
        </View>
      </View>
    </View>
  </View>
</View>
`;
